rpcs3/rpcs3qt/trophy_manager_dialog.cpp

```diff
@@ -212,10 +212,7 @@ trophy_manager_dialog::trophy_manager_dialog(std::shared_ptr<gui_settings> gui_s
 	connect(m_icon_slider, &QSlider::valueChanged, this, [=, this](int val)
 	{
 		m_icon_height = val;
-		if (trophy_slider_label)
-		{
-			trophy_slider_label->setText(tr("Trophy Icon Size: %0x%1").arg(val).arg(val));
-		}
+		trophy_slider_label->setText(tr("Trophy Icon Size: %0x%1").arg(val).arg(val));
```

Comment:
why remove the check?

Reply:
it isn't necessary; its assigned with `new` so it will fail otherwise

Reply:
Well. that's a callback and not not the same scope.
The pointer is just captured.
The change is probably fine, since it's parent is "this" and this is the context of the connect. But idk if the check was added for a reason. So the change is not really worth a potential regression in my opinion.

Reply:
fair enough :)